docs/_docs/developer-guide/scheduler.md

```
@@ -474,7 +599,116 @@ database:
+
+```bash
+cueadmin -show <name> -setSchedulerManaged true
+cueadmin -show <name> -setSchedulerManaged false
+```
+
+#### Hot path: atomic booking Lua
+
+A single Lua script runs the per-frame booking against Redis, executing five updates
+atomically:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)